website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -7,7 +7,7 @@ tags: [Metrics, Semantic Layer]
 ---
 <VersionBlock firstVersion="1.9">
 
-<!-- this whole section is for 1.9 and higher + Release Tracks -->
+<!-- this whole section is for 1.9 and higher + Release tracks -->
```

Comment:
We could probably remove this whole thing since the versionblock is in

Reply:
got it! it was more for any new person coming in 🙏